mesonbuild/scripts/coverage.py

```diff
@@ -3,48 +3,24 @@
 
 from __future__ import annotations
 
-from mesonbuild import environment, mesonlib, coredata
+from mesonbuild import environment, mesonlib
 
 import argparse, re, sys, os, subprocess, pathlib, stat
 import typing as T
 
-def load_coredata(build_dir: str, options: 'argparse.Namespace') -> coredata.CoreData:
-    from mesonbuild.environment import Environment
```

Comment:
Oh never mind, here it's being removed again... if that approach has been abandoned, it should be removed from the patch series.